flybywiresim/aircraft
# fix(a380x): FMS, ECL, BTV fallback fixes

## Summary of Changes


- Fix color of mandatory FMS fields: Not amber anymore when set to 0
- Automatically check DEPARTURE CHANGE ECL as completed, until deferred checklists are implemented
- Don't allow AIRWAYS vertical revisions to be called for airports and runway legs
- Disable CMS vertical revision menu entry for now
- Lower FMS input field auto-blur time from 30s to 20s
- Don't allow FMS input field keystroke if not focused

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions


1. Enter "0" into FUEL&LOAD -> PAX NBR field, value should be cyan not amber
2. Open up ECL in EWD, should show DEPARTURE CHANGE as completed already
3. Try opening up AIRWAYS revisions for airport or runway legs, shouldn't work
4. Click on ALTN RTE field on INIT page and press keys on physical keyboard, shouldn't work


## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page
